pkg/juicefs/juicefs.go

```diff
@@ -379,7 +378,7 @@ func (j *juicefs) JfsUnmount(volumeId, mountPath string) error {
 				delete(j.CacheDirMaps, uniqueId)
 
 				klog.V(5).Infof("Cleanup cache of volume %s in node %s", uniqueId, config.NodeName)
-				go j.processMount.CleanCache(uuid, uniqueId, cacheDirs)
+				go j.processMount.CleanCache(ctx, uuid, uniqueId, cacheDirs)
```

Comment:
cleanCache may take long time, do not set timeout for it.

Reply:
Fixed